pkg/languages/defaults/gradle.go

```diff
@@ -0,0 +1,107 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/draft/pkg/reporeader"
+	log "github.com/sirupsen/logrus"
+)
+
+type GradleExtractor struct {
+}
+
+// GetName implements reporeader.VariableExtractor
+func (*GradleExtractor) GetName() string {
+	return "gradle"
+}
+
+// MatchesLanguage implements reporeader.VariableExtractor
+func (*GradleExtractor) MatchesLanguage(lowerlang string) bool {
+	return lowerlang == "gradle" || lowerlang == "gradlew"
```

Comment:
Also curious if there is any golang gradle parser? Not a blocker to use just curious if a package that could handle that for us exists?

Reply:
I found something few days back - https://github.com/gogradle/gogradle but this repo is archived and not supported. I will keep looking if any exists, please let me know if you find any library to do it. Thanks for the suggestion.